### PR TITLE
Added Bootstrap Styling To All Modals

### LINKED ISSUE #68: Add Styling to LinksForm.jsx
Update `LinksForm.jsx` to use bootstrap css, such that it looks similar to what was done with the Skills modal [here](https://github.com/maryvilledev/skilldirectoryui/pull/50).

**Update:** This should also be done to the TeamMembers modal.

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #68.

### CODE REVIEW

File: src/Skills.jsx [hunk 1]
```diff
@@ -125,11 +125,10 @@ class Skills extends Component {
 
   componentDidMount() {
     const currentId = (this.props.params) ? this.props.params.id : null;
-    if (!currentId) {
+    if (!currentId)
```
Comment (thebho) [inline src/Skills.jsx]: Why did you remove curly braces here?

Comment (Hopding) [reply src/Skills.jsx]: Just to clean it up, since they were not required. I usually avoid adding braces for one-liners out of habit.

Comment (thebho) [reply src/Skills.jsx]: Bad habit, explicit curly braces! http://stackoverflow.com/questions/4797286/are-curly-braces-necessary-in-one-line-statements-in-javascript

Comment (Hopding) [reply src/Skills.jsx]: Fair enough. I'll restore them 👍

**Review by thebho — APPROVED** [review]
